From 7a3d9447f2c12e00b07acafacdabe798edf351eb Mon Sep 17 00:00:00 2001
From: John Tung <watchnight@163.com>
Date: Sun, 21 Aug 2022 11:22:48 +0800
Subject: [PATCH 7/7] fix cpu cgroup mount

---
 libprocessgroup/setup/cgroup_map_write.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libprocessgroup/setup/cgroup_map_write.cpp b/libprocessgroup/setup/cgroup_map_write.cpp
index 17ea06e09..c25bf302a 100644
--- a/libprocessgroup/setup/cgroup_map_write.cpp
+++ b/libprocessgroup/setup/cgroup_map_write.cpp
@@ -217,6 +217,11 @@ static bool SetupCgroup(const CgroupDescriptor& descriptor) {
             result = mount("none", controller->path(), "cgroup", MS_NODEV | MS_NOEXEC | MS_NOSUID,
                            controller->name());
         }
+        if (result < 0 && (!strcmp(controller->name(), "cpu")
+                        || !strcmp(controller->name(), "cpuacct"))) {
+            result = mount("none", controller->path(), "cgroup", MS_NODEV | MS_NOEXEC | MS_NOSUID,
+                           "cpu,cpuacct");
+        }
     }
 
     if (result < 0) {
-- 
2.34.1

