From e83538ff2dff5a9af1cd16a76a1a9efb38e7fcd8 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sun, 9 May 2021 23:29:12 +0800
Subject: [PATCH 3/7] allow override ro.* prop

---
 init/init.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/init/init.cpp b/init/init.cpp
index bf57f50c9..ccba25f8a 100755
--- a/init/init.cpp
+++ b/init/init.cpp
@@ -356,6 +356,8 @@ static void import_kernel_nv(const std::string& key, const std::string& value, b
         strlcpy(qemu, value.c_str(), sizeof(qemu));
     } else if (android::base::StartsWith(key, "androidboot.")) {
         property_set("ro.boot." + key.substr(12), value);
+    } else if (android::base::StartsWith(key, "ro.")) {
+        property_set(key, value);
     }
 }
 
-- 
2.34.1

