From cdc07bdf5ff62c62821070cd11574240ac05212d Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Thu, 24 Jun 2021 11:52:55 +0000
Subject: [PATCH 4/7] disable input subsystem

---
 rootdir/ueventd.rc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index b03d83bf1..0fae2d5e8 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -21,9 +21,6 @@ subsystem msm_camera
     devname uevent_devpath
     dirname /dev/msm_camera
 
-subsystem input
-    devname uevent_devpath
-    dirname /dev/input
 
 subsystem mtd
     devname uevent_devpath
-- 
2.34.1

