From e28cc1e8fddc644f708ed2ca871ab6dfa42fa732 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sun, 27 Jun 2021 09:49:22 +0000
Subject: [PATCH 3/7] ? fix first stage mount

---
 init/first_stage_init.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/first_stage_init.cpp b/init/first_stage_init.cpp
index 649f3ec76..390b40979 100644
--- a/init/first_stage_init.cpp
+++ b/init/first_stage_init.cpp
@@ -403,7 +403,7 @@ int FirstStageMain(int argc, char** argv) {
     }
 
     if (!DoFirstStageMount(!created_devices)) {
-        LOG(FATAL) << "Failed to mount required partitions early ...";
+        LOG(ERROR) << "Failed to mount required partitions early ..."; // HACKED
     }
 
     struct stat new_root_info;
-- 
2.34.1

