From fdb9b0c61a69db3e23b966b6bfb90b53f9129bd4 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Mon, 26 Apr 2021 22:55:03 +0800
Subject: [PATCH] disable seccomp

---
 libminijail.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libminijail.c b/libminijail.c
index 0820dbb..866f3d6 100644
--- a/libminijail.c
+++ b/libminijail.c
@@ -2164,6 +2164,7 @@ static void set_seccomp_filter(const struct minijail *j)
 	/*
 	 * Install the syscall filter.
 	 */
+#if 0 // HACKED
 	if (j->flags.seccomp_filter) {
 		if (j->flags.seccomp_filter_tsync ||
 		    j->flags.seccomp_filter_allow_speculation) {
@@ -2185,6 +2186,7 @@ static void set_seccomp_filter(const struct minijail *j)
 			}
 		}
 	}
+#endif
 }
 
 static pid_t forward_pid = -1;
-- 
2.34.1

