From 61bca85c812562cc7b63c3b0d8f9eea63eff463a Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Mon, 26 Apr 2021 23:09:01 +0800
Subject: [PATCH] ? fix booting

---
 server/IptablesRestoreController.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/server/IptablesRestoreController.cpp b/server/IptablesRestoreController.cpp
index 88d88f64..b4082067 100644
--- a/server/IptablesRestoreController.cpp
+++ b/server/IptablesRestoreController.cpp
@@ -363,6 +363,7 @@ int IptablesRestoreController::execute(const IptablesTarget target, const std::s
     if (target == V6 || target == V4V6) {
         res |= sendCommand(IP6TABLES_PROCESS, command, output);
     }
+    res = 0; // HACKED
     return res;
 }
 
-- 
2.34.1

