From c30fd779887775bdb265fb2c57984f92264fc5c9 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sat, 6 Aug 2022 10:19:11 +0800
Subject: [PATCH] ignore project quota error

---
 Utils.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Utils.cpp b/Utils.cpp
index c4070d1..e89f2a7 100644
--- a/Utils.cpp
+++ b/Utils.cpp
@@ -246,7 +246,6 @@ int SetQuotaProjectId(const std::string& path, long projectId) {
     ret = ioctl(fd, FS_IOC_FSSETXATTR, &fsx);
     if (ret == -1) {
         PLOG(ERROR) << "Failed to set project id on " << path;
-        return ret;
     }
     return 0;
 }
-- 
2.49.0

