From 6357580325e58681df67299b1a2ad33ea3357858 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sat, 26 Jun 2021 05:42:24 +0000
Subject: [PATCH 2/7] allow override ro.* prop

---
 init/property_service.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/init/property_service.cpp b/init/property_service.cpp
index f2606e3..48ada9a 100755
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -1393,6 +1393,8 @@ static void ProcessKernelCmdline() {
     android::fs_mgr::ImportKernelCmdline([&](const std::string& key, const std::string& value) {
         if (StartsWith(key, ANDROIDBOOT_PREFIX)) {
             InitPropertySet("ro.boot." + key.substr(ANDROIDBOOT_PREFIX.size()), value);
+        } else if (StartsWith(key, "ro.")) {
+            InitPropertySet(key, value);
         }
     });
 }
-- 
2.49.0

