From a99b4769df8e0c612051d23ec9f41f31ee00c908 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Mon, 26 Apr 2021 22:55:03 +0800
Subject: [PATCH] disable seccomp

---
 libminijail.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libminijail.c b/libminijail.c
index aab12949..cc9287db 100644
--- a/libminijail.c
+++ b/libminijail.c
@@ -2216,6 +2216,7 @@ static void set_seccomp_filter(const struct minijail *j)
 	/*
 	 * Install the syscall filter.
 	 */
+#if 0 // HACKED
 	if (j->flags.seccomp_filter) {
 		if (j->flags.seccomp_filter_tsync ||
 		    j->flags.seccomp_filter_allow_speculation) {
@@ -2237,6 +2238,7 @@ static void set_seccomp_filter(const struct minijail *j)
 			}
 		}
 	}
+#endif
 }
 
 static pid_t forward_pid = -1;
-- 
2.34.1

