From 5ae6f7c9ec69e701efe41220ce4a7b78db23443f Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sat, 15 Oct 2022 09:12:31 +0800
Subject: [PATCH] ignore compatibility check

---
 VintfObject.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/VintfObject.cpp b/VintfObject.cpp
index 8d9df31e..bd474b4d 100644
--- a/VintfObject.cpp
+++ b/VintfObject.cpp
@@ -635,7 +635,6 @@ int32_t VintfObject::checkCompatibility(std::string* error, CheckFlags::Type fla
                 error->insert(0,
                               "Runtime info and framework compatibility matrix are incompatible: ");
             }
-            return INCOMPATIBLE;
         }
     }
 
-- 
2.34.1

