From 05280bbcc0c2fc80f7bff7c19dad62e6ba7d9b1b Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sat, 15 Oct 2022 09:12:31 +0800
Subject: [PATCH 1/2] ignore compatibility check

---
 VintfObject.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/VintfObject.cpp b/VintfObject.cpp
index f391679..79e0b40 100644
--- a/VintfObject.cpp
+++ b/VintfObject.cpp
@@ -738,7 +738,6 @@ int32_t VintfObject::checkCompatibility(std::string* error, CheckFlags::Type fla
                 error->insert(0,
                               "Runtime info and framework compatibility matrix are incompatible: ");
             }
-            return INCOMPATIBLE;
         }
     }
 
-- 
2.49.0

