From c9c7f6e6471cc2552541fffc432e50b36e02d9f8 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sun, 14 Aug 2022 11:57:11 +0800
Subject: [PATCH 3/3] ignote getTetherStats error

---
 server/TetherController.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/server/TetherController.cpp b/server/TetherController.cpp
index b159d95..a582cd8 100644
--- a/server/TetherController.cpp
+++ b/server/TetherController.cpp
@@ -913,9 +913,11 @@ StatusOr<TetherController::TetherStatsList> TetherController::getTetherStats() {
         }
 
         if (int ret = addForwardChainStats(statsList, statsString, parsedIptablesOutput)) {
+#if 0
             return statusFromErrno(-ret, StringPrintf("failed to parse %s tether stats:\n%s",
                                                       target == V4 ? "IPv4": "IPv6",
                                                       parsedIptablesOutput.c_str()));
+#endif
         }
     }
 
-- 
2.49.0

