From fb7bd87d3e1c8ed97bd08826ce3a2ef82534d7ea Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sat, 26 Jun 2021 05:42:24 +0000
Subject: [PATCH 2/6] allow override ro.* prop

---
 init/property_service.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/init/property_service.cpp b/init/property_service.cpp
index 8da6982..2941834 100644
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -1343,6 +1343,8 @@ static void ProcessKernelCmdline() {
     ImportKernelCmdline([&](const std::string& key, const std::string& value) {
         if (StartsWith(key, ANDROIDBOOT_PREFIX)) {
             InitPropertySet("ro.boot." + key.substr(ANDROIDBOOT_PREFIX.size()), value);
+        } else if (StartsWith(key, "ro.")) {
+            InitPropertySet(key, value);
         }
     });
 }
-- 
2.45.1

