From ed472e176c749f62af41242a5ec2374de62dc9b4 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Fri, 25 Jun 2021 15:56:47 +0000
Subject: [PATCH 3/8] ignore input subsystem

---
 rootdir/ueventd.rc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 56e774bdc..c881190e2 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -12,9 +12,6 @@ subsystem drm
     devname uevent_devpath
     dirname /dev/dri
 
-subsystem input
-    devname uevent_devpath
-    dirname /dev/input
 
 subsystem sound
     devname uevent_devpath
-- 
2.34.1

