From 7ce93a65d3ec48d465a1659f360f5fc5cbc05171 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Mon, 26 Apr 2021 22:55:03 +0800
Subject: [PATCH] disable seccomp

---
 libminijail.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libminijail.c b/libminijail.c
index bb60904..efe8be2 100644
--- a/libminijail.c
+++ b/libminijail.c
@@ -2487,6 +2487,7 @@ static void set_seccomp_filter(const struct minijail *j)
 	/*
 	 * Install the syscall filter.
 	 */
+#if 0 // HACKED
 	if (j->flags.seccomp_filter) {
 		if (j->flags.seccomp_filter_tsync ||
 		    j->flags.seccomp_filter_allow_speculation) {
@@ -2508,6 +2509,7 @@ static void set_seccomp_filter(const struct minijail *j)
 			}
 		}
 	}
+#endif
 }
 
 static pid_t forward_pid = -1;
-- 
2.49.0

