From 17a9b675b0f2cae349440b99acbf74f4da2c3ae2 Mon Sep 17 00:00:00 2001
From: Ziyang Zhou <ziyang.zhou@outlook.com>
Date: Sat, 26 Jun 2021 05:42:24 +0000
Subject: [PATCH 2/8] allow override ro.* prop

---
 init/property_service.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/init/property_service.cpp b/init/property_service.cpp
index 26341b196..a4635054c 100755
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -1269,6 +1269,8 @@ static void ProcessKernelCmdline() {
     ImportKernelCmdline([&](const std::string& key, const std::string& value) {
         if (StartsWith(key, ANDROIDBOOT_PREFIX)) {
             InitPropertySet("ro.boot." + key.substr(ANDROIDBOOT_PREFIX.size()), value);
+        } else if (StartsWith(key, "ro.")) {
+            InitPropertySet(key, value);
         }
     });
 }
-- 
2.34.1

